packages/shared/src/components/icons/TimeZone/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './DaytimeIcon';
+export * from './NighttimeIcon';
```

Comment:
These files were just moved and turned to how we normally use SVGs. Was getting an error due to some existing implementations utilizing the icon props.